src/Models/basket/Transformers/BasketLineTransformer.ts

```diff
@@ -18,7 +19,7 @@ export default class BasketLineTransformer extends BaseTransformer {
             productVariant: this.includeProductVariant(basketLine),
         });
     }
-    includeProductVariant(basketLine: APIBasketLineInterface): Array<BasketLine> {
+    includeProductVariant(basketLine: APIBasketLineInterface): ProductVariant {
         return this.collection(basketLine, 'product_variant', new ProductVariantTransformer());
```

Comment:
A collection will return an array, but you're returning a single ProductVariant. Should this be `this.item()` not collection?